src/scene.rs

```diff
@@ -204,6 +237,9 @@ impl AmbientLight {
     }
 }
 
+/// The light source that illuminates all objects equally from a given direction,
+/// like an area light of infinite size and infinite distance from the scene;
```

Comment:
So, only "The light source that illuminates all objects equally from a given direction"?